### PR TITLE
refactor: remove swagger route and docs annotations (DEV-1335)

### PR DESCRIPTION
**Pull request by mpro7**
Issue Number: DEV-1335

### Basic Requirements

Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

### PR Type

What kind of change does this PR introduce?

- [ ] Bugfix: represents bug fixes
- [x] Refactor: represents production code refactoring
- [ ] Feature: represents a new feature
- [ ] Documentation: documentation changes (no production code change)
- [ ] Chore: maintenance tasks (no production code change)
- [ ] Style: styles updates (no production code change)
- [ ] Test: all about tests: adding, refactoring tests (no production code change)
- [ ] Other... Please describe:

### Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### Does this PR change client-test-data?

- [ ] Yes (don't forget to update the JS-LIB team about the change)
- [x] No

### CODE REVIEW

File: docs/Readme.md [hunk 1]
```diff
@@ -9,7 +9,6 @@ The `src` folder contains the following documentation sources:
 
 - `src/api-v1`: The DSP-API V1 (JSON) Request and Response Format documentation. Source can be found in `salsah1/src/typescript_interfaces`
 - `src/api-v2`: The DSP-API V2 (JSON-LD) Request and Response Format documentation.
-- `src/api-admin`: The DSP-API Admin (JSON) Request and Response format Swagger-based documentation.
```
Comment (irinaschubert) [inline docs/Readme.md:12]: This whole structure is not up-to-date. Could you go over the whole `Readme.md` file if you touch it anyway?

**Review by irinaschubert — APPROVED** [review]
LGTM

File: docs/Readme.md (hunk 1 above)
Comment (mpro7) [reply docs/Readme.md]: Although it's out of the scope of this PR I cleaned it up. There should be another task defined to clean the whole API documentation though.

Comment (mpro7) [reply docs/Readme.md]: And I removed too much :)
Do we actually need this "test"?
```
  test-docs-build:
    name: Test docs
    runs-on: ubuntu-latest
    steps:
      - name: Checkout source
        uses: actions/checkout@v3
      - name: Install requirements (pip, npm, apt)
        run: |
          python -m pip install --upgrade pip
          pip install -r docs/requirements.txt
          npm install --global typedoc
          sudo apt-get install graphviz
      - name: Build docs
        run: make docs-build
```

Comment (irinaschubert) [reply docs/Readme.md]: Yes, we need it. It's to make sure that the docs compile.

**Pull request merged**